sql/drop.go

```diff
@@ -0,0 +1,48 @@
+package sql
+
+import (
+	"fmt"
+
+	"github.com/cockroachdb/cockroach/keys"
+	"github.com/cockroachdb/cockroach/sql/parser"
+	"github.com/cockroachdb/cockroach/structured"
+)
+
+// DropTable drops a table.
+func (p *planner) DropTable(n *parser.DropTable) (planNode, error) {
+	//TODO(XisiHuang): should do truncate and delete descripter in the same txn
```

Comment:
more nits: s/descripter/descriptor/ and probably wanna set up your editor to rewrap long comment lines (unless you haven't already).